registry.k8s.io/images/k8s-staging-kind/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS file documentation:
+# https://github.com/kubernetes/kubernetes/blob/master/docs/devel/owners.md
```

Comment:
cloud provider kind is a distinct subproject and should be published via it's own staging?

Reply:
too late now I guess, but we should be careful about this part in the future, with immutable images we're stuck with naming and we have arbitrary nesting so we could do distinct paths for each.

Reply:
... not too late